### PR TITLE
clarify the phrase 'in-memory' and fix a typo

### PR DESCRIPTION
**Pull request by sefabey**
- clarified what we mean by in-memory. This will be helpful for social scientists who might not be familiar with the concept.
- fixed a typo

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

### CODE REVIEW

**Comment by atheobold** [comment]
@sefabey thank you for this contribution! I think it's a great idea to provide learners with an explanation of what we mean by "memory". I added a tiny bit extra that clarifies what RAM is used for and how it differs from CDs/DVDs. Also, good catch on the typo! It helped me find another typo!

File: _episodes_rmd/.ignore-05-databases.Rmd [hunk 1]
```diff
@@ -28,9 +28,14 @@ source("../bin/download_data.R")
 
 ## Introduction
 
-A common problem with R in that all operations are conducted in-memory and thus
-the amount of data you can work with is limited by available memory. So far, we have used small datasets
-that can easily fit into your computer's memory. But what about datasets that are too large for your
+A common problem with R in that all operations are conducted in-memory. Here, "memory" means a computer 
+drive called Random Access Memory (RAM for short) which is phsically installed on your computer. RAM 
+allows for data to be read in in nearly the same amount of time regardless of its physical location
```
Comment (kelseygonzalez) [inline _episodes_rmd/.ignore-05-databases.Rmd:33]: duplicate "in in"

**Comment by kelseygonzalez** [comment]
@sefabey Thanks for the great contribution. This is very clear and well written. As with @atheobold , I fixed another typo I found ("in in") and merged the PR.

File: _episodes_rmd/.ignore-05-databases.Rmd (hunk 1 above)
Comment (atheobold) [reply _episodes_rmd/.ignore-05-databases.Rmd]: I agree that this sentence was awkward, my bad. I was attempting to say that the data is being "read in" / imported _in_ nearly the same amount of time.

**Pull request merged**